avocado_virt/plugins/virt_bootstrap.py

```diff
@@ -82,9 +83,10 @@ def run(self, args):
                           'it (205 MB). Please wait...', jeos_dst_path)
             else:
                 LOG.debug('JeOS at %s is either corrupted or outdated. '
-                          'Downloading a new copy (205 MB). '
+                          'Downloading a new copy (hundreds of MBs). '
```

Comment:
Like this? https://xkcd.com/612/

Reply:
Well, it's more accurate than 205...

Reply:
I see the point for the change (less maintenance) but that file is 205 MB:

```
$ ls -lh ~/avocado/data/images/
total 1.2G
-rw-rw-r--. 1 cleber cleber 939M Feb 16 12:25 jeos-25-64.qcow2
-rw-rw-r--. 1 cleber cleber 205M Feb 16 12:25 jeos-25-64.qcow2.xz
```

Reply:
OK, let's keep it this way (I saw it in kb and haven't re-calculate it especially when embedding size in code looks odd to me). Anyway adding this to https://trello.com/c/SoIfTMHO/1245-avocado-virt-make-jeos-version-configurable-instead-of-hardcoded-valueslinks and removing from here.